EarlGrey/Common/GREYAppleInternals.h

```diff
@@ -120,6 +120,28 @@ IOHIDEventRef IOHIDEventCreateDigitizerFingerEvent(CFAllocatorRef allocator,
 - (void)_clearTouches;
 @end
 
+/**
+ *  A private class that represents backboard services accelerometer.
+ */
+@interface BKSAccelerometer : NSObject
+
+@property (nonatomic) BOOL accelerometerEventsEnabled;
```

Comment:
document?

Reply:
What's more descriptive than the name of the property?

Reply:
@LeoNatan we use a small description for each property / method / class for doxygen purposes. Thanks.

Reply:
Here it's not so much for dOxygen than it is for someone trying to understand how this private API works. Maybe setting this to YES needs you to call another method or reading from it isn't valid until something else is called...that's why I always add some documentation over it...feel free to skip if it you feel it's intuitive.

Reply:
Fortunately, for that property specifically the getter and setter were just default synthesizer names. Otherwise, I would have imported them as methods instead of the property itself. I added a small comment.